internal/bundlereader/loaddirectory_test.go

```diff
@@ -344,6 +344,91 @@ func TestGetContent(t *testing.T) {
 	}
 }
 
+func TestGetContentOCI(t *testing.T) {
+	cases := []struct {
+		name    string
+		source  string
+		version string
+
+		result      []string
+		expectedErr string
+	}{
+		// Note: These tests rely on external hosts and DNS resolution.
```

Comment:
Alternatively I can change the code, so I can mock the registry client. I'd then delete these test (because they mostly test the helm code) and if the client is initialized and if error wrapping works.